toolkits/landcover/impl/dataset.js

```diff
@@ -33,6 +33,19 @@ var Dataset = function(collection, defaultVisParams) {
   this.defaultVisParams_ = defaultVisParams;
 };
 
+/**
+ * Apply a filter to this dataset.
+ *
+ * @param {!ee.Filter} filter A filter to apply to this dataset.
+ * @return {!Dataset}
+ */
+Dataset.prototype.filter = function(filter) {
+  var args = NamedArgs.extractFromFunction(Dataset.prototype.filterDate, arguments);
```

Comment:
Should "Dataset.prototype.filterDate" be "Dataset.prototype.filter"?

Reply:
Yikes, thanks, good catch.